footil/tests/test_xcollections.py

```diff
@@ -0,0 +1,34 @@
+from footil import xcollections
+
+from . import common
+
+
+class TestCollections(common.TestFootilCommon):
+    """Common class for collections module tests."""
+
+    @classmethod
+    def setUpClass(cls):
+        """Set up data for collection helper test cases."""
+        super().setUpClass()
+        cls.sequence = [1, 2, 3, 4, 5]
+
+    def test_01_batch(self):
```

Comment:
As these tests can be done independently and are very fast, its better to split into multiple methods for each case. 

Also that helper is just mixing method call for test and then testing result. It Would be better to not group `set up` and results. I don't think you even need any helper as there is nothing to automate. With all those helpers, you have just written more code than you have to. You should do instead:

```python
self.assertEqual(xcollections.batch([1, 2, 3], 2), [[1, 2], [3])
```